tests/Unit/Css/StyleRuleTest.php

```diff
@@ -93,7 +94,9 @@ public function getDeclarationAsTextReturnsConcatenatedDeclarationsFromRules(
         $declarationBlock = new DeclarationBlock();
         foreach ($declarations as $declaration) {
             $rule = new Rule($declaration['property']);
-            $rule->setValue($declaration['value']);
```

Comment:
I guess this worked because the test only ends up doing an implicit `__tostring()` on the value in `Rule::render()`, so we got away with providing a string instead of a `Value`.